### PR TITLE
remove windows releases

### PR DESCRIPTION
**Pull request by henry739**
re AB#9701

### CODE REVIEW

**Review by robinbryce — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A general familiarity with _verifiable data structures_, and in particular binar
 
 ## Installation
 
-Veracity provides native binaries for [Mac OS](#mac-install), [Linux](#linuxwsl-install) and Windows on the [releases](https://github.com/datatrails/veracity/releases) page.
+Veracity provides native binaries for [Mac OS](#mac-install) and [Linux](#linuxwsl-install) on the [releases](https://github.com/datatrails/veracity/releases) page.
```
Comment (robinbryce) [inline README.md:9]:
```suggestion
Veracity provides native binaries for [Mac OS](#mac-install), [Linux](#linuxwsl-install) and Windows (WSL)
```

**Review by robinbryce — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ A general familiarity with _verifiable data structures_, and in particular binar
 
 ## Installation
 
-Veracity provides native binaries for [Mac OS](#mac-install), [Linux](#linuxwsl-install) and Windows on the [releases](https://github.com/datatrails/veracity/releases) page.
+Veracity provides native binaries for [Mac OS](#mac-install), [Linux](#linuxwsl-install) and Windows (WSL) on the [releases](https://github.com/datatrails/veracity/releases) page.
```
Comment (j-hartley) [inline README.md:9]: I don't think we should say that

**Review by j-hartley — APPROVED** [review]

**Pull request merged**